scripts/us_census/acs5yr/subject_tables/common/resolve_geo_id.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
please add license to both files -- see other files in repo for example

Reply:
Thank you for the review. I have checked in the files with the license header

Reply:
Force push to clean the head, I added a few scripts that are meant to be in the subsequent PR by mistake to the branch linked with this PR.